sjb/config/test_cases/ami_build_origin_int_rhel_base.yml

```diff
@@ -28,6 +28,10 @@ actions:
     script: |-
       oct prepare repositories
   - type: "script"
+    title: "install mvn and java"
+    script: |-
+      sudo yum install -y maven java-1.8.0-openjdk-devel
```

Comment:
lgtm but i have to wonder if @stevekuznetsov is installing packages via another mechanism since this is the only instance of this in this job